.travis.yml

```diff
@@ -1,39 +1,46 @@
-sudo: false
-services:
-- docker
+dist: xenial
 language: python
 cache: pip
-services:
+service:
 - docker
 
-before_install:
-- pyenv versions
-- pyenv version-name
-- env
-
 install:
-- pip install tox
+- env | sort
 
 python:
 - '2.7'
 - '3.4'
 - '3.5'
 - '3.6'
-- '3.7-dev'
+- '3.7'
+- '3.8-rc'
 
 env:
-- SALT=-v2018.3 BACKEND=-cherrypy CODECOV=py
-- SALT=-v2018.3 BACKEND=-tornado CODECOV=py
-- SALT=-v2019.2 BACKEND=-cherrypy CODECOV=py
-- SALT=-v2019.2 BACKEND=-tornado CODECOV=py
+- SALT=-v2018.3 BACKEND=-cherrypy
+- SALT=-v2018.3 BACKEND=-tornado
+- SALT=-v2019.2 BACKEND=-cherrypy
+- SALT=-v2019.2 BACKEND=-tornado
```

Comment:
we do not want to remove CODECOV, instead we should be setting this up to also run flake8 tests.